app/controllers/events_controller.rb

```diff
@@ -26,12 +26,24 @@ def create
   end
 
   def show
-    @event = Event.find_by_id(params[:id])
     add_breadcrumb @event.title
     @markers = BuildMarkersWithInfoWindow
                    .with(Event.build_by_coordinates([@event]), self)
   end
 
+  def edit
+    @current_user = current_user
```

Comment:
I can't immediately see why this is needed - am I missing something? :-)

Reply:
I think I was using it in the views, but then changed it? I'll have a closer look, but that is my guess.